src/main/java/com/gw/exercises/task1/multsAndDivids.java

```diff
@@ -0,0 +1,25 @@
+package com.gw.exercises.task1;
+
+public class multsAndDivids {
+
+    public static void main(String[] args) {
+        //        Напиши програму яка ділить два числа ( 36 на 13);
+        System.out.println(divider(36, 13));
+
+        //        Напиши програму яка виводить на екран результат наступної дії ( a + b * c), де а = -5; b = 8; c =6;
+        int a = -5, b = 8 , c = 6;
```

Comment:
Don't write all variables in one line.
One variable = one new line with initialization.